xwiki-platform-core/xwiki-platform-filter/xwiki-platform-filter-instance/xwiki-platform-filter-instance-document/pom.xml

```diff
@@ -40,4 +40,28 @@
       <version>${project.version}</version>
     </dependency>
   </dependencies>
+  <build>
+    <plugins>
+      <plugin>
+        <groupId>org.revapi</groupId>
+        <artifactId>revapi-maven-plugin</artifactId>
+        <configuration>
+          <analysisConfiguration>
+            <revapi.differences>
```

Comment:
We don't put revapi ignores there, every ignore is going to `xwiki-platform-core` pom.xml: the reason is that we get them in the RN so we need them to be all at the same place.

Reply:
Also I think we don't use `ignore` anymore but we put a criticality see the example: https://github.com/xwiki/xwiki-platform/blob/master/xwiki-platform-core/pom.xml#L107

Reply:
See also: https://dev.xwiki.org/xwiki/bin/view/Community/DevelopmentPractices#HRevapiIgnores

Reply:
Oups, it seems that's for me to fix :)

Reply:
Ok you can rebase and fix the conflict you should be able to test it.

Reply:
> [ERROR] - Component [org.xwiki.configuration.internal.PermanentConfigurationSource] is not declared in [/home/raph/dl/xwiki/xwiki-platform/xwiki-platform-core/xwiki-platform-configuration/xwiki-platform-configuration-default/target/classes/META-INF/components.txt]! Consider adding it or if it is normal use the "staticRegistration" parameter as in "@Component(staticRegistration = false)"

Definitely unrelated but also not true so it's strange you get that.